src/operation.rs

```diff
@@ -20,6 +20,7 @@ pub struct Operation {
     pub description: Option<String>,
     /// Additional external documentation for this operation.
     #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(rename(serialize = "externalDocs"))]
     pub external_documentation: Option<ExternalDocumentation>,
```

Comment:
@glademiller what would you think about renaming this field and then bumping the version number of the crate?

Reply:
@dianezhou96